app/internal/wm/GioView.java

```diff
@@ -132,6 +134,64 @@ private void setCursor(int id) {
 		setPointerIcon(pointerIcon);
 	}
 
+    private void setBarColor(String t, int color, int luminance) {
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP) {
+            return;
+        }
+
+        Window window = ((Activity) this.getContext()).getWindow();
+        if (window == null) {
+            return;
+        }
+
+        int insetsMask = 0;
+        int viewMask = 0;
+
+        if (t == "Status") {
```

Comment:
This won't (shouldn't) work; you need to do `t.equals("Status")`. However, let's use the typesafer enums instead of free-form strings to distinguish the cases.

Reply:
It works. But, I replace with enums. 👍